lib/Prelude.fram

```diff
@@ -43,3 +43,17 @@ pub module Int64
   pub let one  = 1L
   pub let ofInt (n : Int) = n.toInt64
 end
+
+pub let assert condition msg = 
```

Comment:
Seems like this PR is up so long, that this will be replaced by `assert` from #158